common/lib/capa/capa/xqueue_interface.py

```diff
@@ -15,6 +15,8 @@
 
 # Wait time for response from Xqueue.
 XQUEUE_TIMEOUT = 35  # seconds
+CONNECT_TIMEOUT = 3.05
```

Comment:
actually python request module also using same name as used in for timeout params. Moreover, requests module also considering timeouts in seconds as mentioned below:
http://docs.python-requests.org/en/master/user/advanced/#timeouts
I think there is no need to mention it